### PR TITLE
Issue#1005 web api examples editor (#1142)

### PR DESCRIPTION
**Pull request by schalkneethling**
@wbamberg @hobinjk This cleans up the interactive examples editor repo and switches to using BoB for building the interactive examples. This looks clean, and all examples tested works. r?

### CODE REVIEW

**Comment by schalkneethling** [comment]
I am going to let this one steam a bit and click through some more examples. If I get no objections, I will most likely merge this tomorrow :tada:

File: CONTRIBUTING-CSS.md [hunk 1]
```diff
@@ -189,18 +188,17 @@ The `title` property is displayed above the editor, and should be of the form: "
...
 So the meta.json entry for a function would look like:
 
 ```
 "translateX": {
-    "baseTmpl": "tmpl/live-css-tmpl.html",
     "cssExampleSrc":
         "../../live-examples/css-examples/transforms/translate.css",
     "exampleCode":
-        "live-examples/css-examples/transforms/function-translateX.html",
+        "./live-examples/css-examples/transforms/function-translateX.html",
```
Comment (wbamberg) [inline CONTRIBUTING-CSS.md]: It looks like the `cssExamplesSrc` line should also be updated, here and in the other spots in this file (see https://github.com/mdn/interactive-examples/pull/1174/files#diff-354f523859ee264d879542cab1eb4f71R4).

**Comment by schalkneethling** [comment]
@wbamberg Issue #1175, are you seeing that against current production?

**Comment by schalkneethling** [comment]
@wbamberg Thanks for all the review. I am going to be overly cautious and merge this first thing tomorrow my local time. 🎉

**Pull request merged**